src/style.css

```diff
@@ -1,3 +1,55 @@
 body {
+  font-family: Roboto, sans-serif;
   margin: 0;
 }
+
+header {
+  display: flex;
+  padding: 0 50px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo__img {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.nav__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.nav__link {
+  display: block;
+  position: relative;
+  height: 60px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. use padding instead